index.html

```diff
@@ -60,8 +60,25 @@
   <section id="introduction">
     <h2>Introduction</h2>
     <p>
-      TODO(henbos): Fill in.
+      This document contains proposed extensions to the [[WEBRTC]] specification,
+      where it was thought inappropriate to incorporate them directly into that specification
+      at the point in time when they were written, but they were too small to warrant
+      writing a separate document.
+    </p>
+    <p>
+      If, at a later time, it is a good time to add them to the base specification, that
+      will be done. Deciding factors will include maturity of the extension, consensus on
```

Comment:
Too much passive language for my taste. I don't think we need to explain. How about just:

This document contains proposed extensions to the [[WEBRTC]] specification. These extensions may be added to the base specification should they reach maturity in implementation and gain consensus.